scripts/gWMS-CMSRunAnalysis.sh

```diff
@@ -89,25 +184,27 @@ END
         fi
     fi
     echo "======== Finished condor_chirp -ing the exit code of the job. Exit code of condor_chirp: $CHIRP_EC ========"
+
 }
 
-echo "======== gWMS-CMSRunAnalysis.sh STARTING at $(TZ=GMT date) on $(hostname) ========"
-echo "User id:    $(id)"
-echo "Local time: $(date)"
-echo "Hostname:   $(hostname -f)"
-echo "System:     $(uname -a)"
-echo "Arguments are $@"
+function parse_cmsrun {
 
-# In the future, we could use condor_chirp to add a classad which records which
-# is the current status of a job. 
-# For example, we could set "bootstrap", "running", "stageout", "finished"
-# We do not do it because we currently have no use for it.
-#echo "======== Attempting to notify HTCondor of job bootstrap ========"
-#condor_chirp set_job_attr_delayed Chirp_CRAB3_Job_Phase bootstrap
+    echo "======== Parsing output of cmsRun at $(TZ=GMT date)========"
```

Comment:
you to not really parse the output (root files) !!
as usual, please one, max 2 lines indicating what this function should do

Reply:
Bad name indeed, i renamed it to `check_cmsrun` and improved the comment a bit

Reply:
we do not only run cmsRun !!